source/_components/switch.kankun.markdown

```diff
@@ -30,14 +30,40 @@ switch:
       host: hostname_or_ipaddr
 ```
 
-Configuration variables:
-
-- **switches** (*Required*): The array that contains all Kankun switches.
-  - **identifier** (*Required*): Name of the Kankun switch as slug. Multiple entries are possible.
-    - **host** (*Required*): Hostname or IP address of the switch on the local network.
-    - **name** (*Optional*): Friendly name of the switch.
-    - **port** (*Optional*): HTTP connection port, defaults to 80.
-    - **path** (*Optional*): Path of CGI script, defaults to `/cgi-bin/json.cgi`.
-    - **username** (*Optional*): Username for basic authentication.
-    - **password** (*Optional*): Password for basic authentication.
-
+{% configuration %}
+switches:
+  description: The array that contains all Kankun switches.
+  required: true
+  type: map
+  keys:
+    identifier:
+      description: Name of the Kankun switch as slug. Multiple entries are possible.
+      required: true
+      type: string
+    host:
```

Comment:
this is incorrect... `host` (and all the others below) are options of a unique device and are therefore nested deeper.

Reply:
so the type of identifier has to be a map and then put everything under it in those keys?

Reply:
This is the confusing part of the documentation at this point. We are still uncertain about a permanent solution. Until then, we keep it this way.